### DISCUSSION TITLE
Can't connect Host management (Kali) to Security Onion

### DISCUSSION THREAD

**Discussion by vesselloJunior** [Security-Onion-Solutions/securityonion#11248] [Unsupported Versions] [answered]
I'm having a difficulty to connecting my host management VM (Kali) to the security Onion.

I added the HM (Kali) IP address via so-allow but still can't communicate both VMs. 

Both VMs are the same network, what am I doing wrong? Send help please

**Comment by dougburks** [comment, marked as answer]
Based on your tweet at https://twitter.com/EsselSebastian/status/1698976153581584476, it looks like you've resolved this issue so I'm closing it.
